src/kerberos/mod.rs

```diff
@@ -769,8 +769,16 @@ impl<'a> Kerberos {
                     #[cfg(feature = "scard")]
                     CredentialsBuffers::SmartCard(smart_card) => {
                         let pin = utf16_bytes_to_utf8_string(smart_card.pin.as_ref()).into_bytes();
+                        warn!(?pin, pin2 = ?smart_card.pin);
```

Comment:
question: Is it intended to be a warning? This sounds like reaching this code path is not great.

Reply:
It's a leftover. I thought I removed it...
Fixed.